ui/app/components/secret-engine/configure-aws.ts

```diff
@@ -146,6 +162,20 @@ export default class ConfigureAwsComponent extends Component<Args> {
     this.args.leaseConfig.unloadRecord();
   }
 
+  @action
+  onChangeAccessType(accessType: string) {
+    this.accessType = accessType;
+    const { rootConfig } = this.args;
```

Comment:
TIL (or can't figure it out) but I was unable to destructure attributes and use them below because they're within if statements and I think javascript doesn't like that.